tools/tenscan/frontend/src/lib/siteMetadata.ts

```diff
@@ -2,15 +2,65 @@ import { socialLinks } from "./constants";
 
 export const siteMetadata = {
   companyName: "Tenscan",
-  metaTitle:
-    "Ten, a decentralized Layer 2 Rollup protocol designed to hyper-scale and encrypt the Ethereum blockchain.",
+  metaTitle: "Tenscan - Real-Time Blockchain Explorer",
   description:
-    "Tenscan allows you to explore and search activities taking place on the Ten Layer 2 blockchain",
-  siteUrl: "https://ten.xyz",
-  siteLogo: ``,
-  siteLogoSquare: ``,
-  email: "team@obscu.ro",
+    "The privacy-preserving blockchain explorer for TEN. View transaction history and smart contract interactions without compromising sensitive data. Experience the future of blockchain transparency.",
```

Comment:
We want to stay away from the words privacy/private - can we change "The privacy-preserving blockchain explorer for TEN" to "TenScan allows you to explore and search the TEN blockchain"

Reply:
Maybe we update this too, @CaisManai? https://github.com/ten-protocol/go-ten/blob/56fee0ee2e862f2c6d5b978a2f1956ec58ff80b3/design/security/Source_of_truth.md?plain=1#L48

Reply:
Also, is it `Tenscan` or `TenScan`? @CaisManai

Reply:
TENScan, I believe